xml/metering-metering_overview.xml

```diff
@@ -9,24 +9,16 @@
  <title>Metering Service (Ceilometer) Overview</title>
  <para>
    The &productname; metering service collects and provides access to OpenStack
-   usage data that can be used for billing reporting such as showback, and
+   usage data that can be used for billing reporting such as showback and
    chargeback. The metering service can also provide general usage reporting.
-   Ceilometer acts as the central collection and data access service to
-   the meters provided by all the OpenStack services. The data collected
-   is available both through the Monasca API and the Ceilometer V2 API.
- </para>
- <important>
-   <para>
-     Ceilometer V2 API was deprecated in the Pike release upstream.  Although
-     the Ceilometer V2 API is still available with &productname;, to prepare for
-     eventual removal of Ceilometer V2 API, we recommend that users switch to
-     the Monasca API to access data.
+   Ceilometer acts as the central collection and data access service to the
```

Comment:
Lost the indentation here.  I assume that doesn't matter for the xml, just looked odd that it now lines up with <para> instead of being two spaces right of it.

Reply:
strange...looks okay in the source file.